docs/resources/Invite.md

```diff
@@ -2,15 +2,15 @@
 
 ### Invite Object
 
-Represents a code that when used, adds a user to a guild or group DM channel.
+Represents a code that when used, adds a user to a guild or group DM channel or adds a user to the inviter's friends list or opens a DM if they are already friends.
 
 ###### Invite Structure
 
 | Field                       | Type                                                                                | Description                                                                                                                       |
 |-----------------------------|-------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
 | code                        | string                                                                              | the invite code (unique ID)                                                                                                       |
 | guild?                      | partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object                          | the guild this invite is for                                                                                                      |
-| channel                     | partial [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object                    | the channel this invite is for                                                                                                    |
+| channel                     | nullable partial [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object           | the channel this invite is for                                                                                                    |
```

Comment:
Should be `?partial channel object` to be consistent with the rest of the docs.

This is still technically in experiment, which sucks, because this made a breaking API change. I'll talk to the team running the experiment and see where it's at.